src/ccrec/models/bert_mt.py

```diff
@@ -198,4 +198,4 @@ def bmt_main(item_df, expl_response, gnd_response, max_epochs=50, alpha=0.05, be
     reranking_scores = bmt.transform(gnd) + gnd.prior_score
     metrics = rime.metrics.evaluate_item_rec(gnd.target_csr, reranking_scores, 1)
 
-    return metrics, reranking_scores, bmt
+    return metrics, reranking_scores, bmt
```

Comment:
line break?

Reply:
Let's add one line break at the end of each file. This makes the files prettier.